.github/workflows/tests.yml

```
@@ -0,0 +1,40 @@
+name: UnitTests
+on: 
+  pull_request:
+    branch: 
+      - '*'
+  push:
+    branch:
+      - '*'
+
+jobs:
+  rpmlinter:
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v2
+    - name: rpmlinter
+      run:  make spec && docker run -v "$PWD"/python-boto.spec:/specs/python-boto.spec:z ubun/rpmlinter
+
+  python_test:
+    runs-on: self-hosted
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [2.7,3.6]
+    env:
+      BOTO_CONFIG: '/tmp/nowhere'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+      env:
+        AGENT_TOOLSDIRECTORY: '/opt/hostedtoolcache'
+    - name: Install Python ${{ matrix.python-version }} Dependencies    
+      run: |
+        if [ ${{ matrix.python-version }} == 2.7 ]; then python -m pip install -r requirements-py26.txt; fi
+        python -m pip install -r requirements.txt
```

По идее такие выражения лучше писать со встроенным `if` синтаксисом:
```suggestion
      run: |
        python -m pip install -r requirements-py26.txt
        python -m pip install -r requirements.txt
       if: ${{ matrix.python-version == '2.7' }}
```

---

тут еще для 2.7 питона ставятся зависимости 2.6. это не ошибка?